docker/Dockerfile-ubuntu-20.04

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:20.04 as base
+
+RUN apt-get update \
+    && DEBIAN_FRONTEND="noninteractive" TZ="America/New_York" \
+        apt install -y git clang wget curl autoconf \
+        libglib2.0-dev libboost-dev libkrb5-dev libsystemd-dev libssl-dev \
+        libboost-program-options-dev libboost-filesystem-dev byacc make
+
+RUN cd /root && git clone https://github.com/Kitware/CMake.git -b release \
+    && cd CMake && ./configure && make -j4 &&  pwd && make install
+
+RUN cd /root && git clone https://github.com/krb5/krb5.git -b krb5-1.21.2-final \
+   && cd krb5/src && autoconf && autoreconf && ./configure && make -j4 && make install
+
+RUN cd /root && git clone --recurse-submodules -b v1.58.0 https://github.com/grpc/grpc \
+    && cd grpc/ && mkdir build && cd build \
+    && cmake -DgRPC_INSTALL=ON -DgRPC_BUILD_TESTS=OFF -DCMAKE_CXX_STANDARD=17 ../ \
+    && make -j4 \
+    && make install
+
+RUN apt-get install wget -y && wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb \
+    && dpkg -i packages-microsoft-prod.deb \
+    && rm packages-microsoft-prod.deb \
+    && apt-get update \
+    && apt-get install -y dotnet-sdk-6.0 \
+    && ln -s '/usr/share/dotnet' '/usr/lib/dotnet'
+
+RUN cd /root && git clone https://github.com/fordth/credentials-fetcher \
```

Comment:
When do we plan to update URL to point to official repository?

Reply:
The PR has been updated with the correct repository